slack.go

```diff
@@ -33,10 +34,24 @@ func sendToSlack(message SlackMessage) error {
 	if err != nil {
 		return err
 	}
-	_, err = http.Post(slackWebHook, "application/json", bytes.NewBuffer(buf))
+	resp, err := http.Post(slackWebHook, "application/json", bytes.NewBuffer(buf))
+
 	if err != nil {
+		log.Printf("Error sending message to slack  %s", err.Error())
 		return err
 	}
 
+	defer resp.Body.Close()
+
+	if resp.StatusCode > 299 {
+		bodyBytes, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			log.Fatal(err)
+		}
+		bodyString := string(bodyBytes)
+
+		log.Printf("Error sending message to slack. StatusCode: %d. Resp: %s", resp.StatusCode, bodyString)
```

Comment:
Can we actually make this an error `fmt.errorf` and return it upstream so the command exits with a non 0 code?

Reply:
Yeah, happy to make the change.

Reply:
Done.